libs/ks-clarity/ks-clr-button/src/lib/buttons.component.html

```diff
@@ -1,38 +1,15 @@
-<h1>Buttons & Button Groups</h1>
-<!-- <prio-button [btn_type]="'primary'" [btn_text]="'Primary'" (prio_click)="print($event)">
-</prio-button> -->
-
-<!-- <prio-button>
-  <button class="btn btn-primary" (click)="print($event)">primary</button>
-</prio-button>
-<prio-button>
-  <button class="btn btn-secondary" (click)="print($event)">secondary</button>
-</prio-button>
-<prio-button>
-  <button class="btn btn-success" (click)="print($event)">success</button>
-</prio-button>
-<prio-button>
-  <button class="btn btn-warning" (click)="print($event)">warning</button>
-</prio-button>
-<prio-button>
-  <button class="btn btn-danger" (click)="print($event)">danger</button>
-</prio-button><br> -->
-<prio-button>
-  <button [clrLoading]="validateStates[0]" class="btn btn-primary" (click)="validateDemoCustomized(validateStates[0],0)">Validate</button>
+<h1>Custom Button</h1>
+<!-- Default prio button -->
+<prio-button [label]="'Default'" (Click)="onClick($event)">
 </prio-button>
-<prio-button>
-  <button [clrLoading]="validateStates[1]" class="btn btn-secondary" (click)="validateDemoCustomized(validateStates[1],1)">Validate</button>
+<!-- Customized prio button -->
+<prio-button [loadingState]="loadingState" [type]="'btn-success'"
```

Comment:
There is already an attribute called `type` in the html button element so we should untouch default html elements apis and use custom one with prefixes like `clr-type` or `btn-type`